src/modules/UI/Settings/selectors.js

```diff
@@ -25,7 +25,12 @@ export const getExchangeTimer = (state: State): number => {
 
 export const getCurrencySettings = (state: State, currencyCode: string) => {
   const settings = getSettings(state)
-  const currencySettings = settings[currencyCode] || isoFiatDenominations[currencyCode]
+  let currencySettings
+  if (settings[currencyCode] || isoFiatDenominations[currencyCode]) {
+    currencySettings = settings[currencyCode] || isoFiatDenominations[currencyCode]
+  } else {
+    currencySettings = settings.customTokens.find((token) => token.currencyCode === currencyCode)
```

Comment:
why are you treating user added tokens different that the default enabled tokens?

Reply:
If you look at the data structure for the settings in Redux, the denomination keys (eg `REP`, `WINGS`, etc) are mixed in with the rest of the top-level settings which, in my opinion, is a major issue since it can eventually cause namespace clashes with actual settings. If someone happened to add a token with a currency code like `pinMode` or `PINMODE`, there's a chance something bad could happen. So I am storing them separately and in the format that matches other currencyInfo's on the wallet objects.